### PR TITLE
Update use cases for DG

### LINKED ISSUE #241: Update use cases in DG

### PR DESCRIPTION
**Pull request by javinchua**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -489,51 +496,58 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **MSS**
 
 1.  User requests to create an internship entry with the necessary details.
-2.  InternshipBook adds the internship entry to the list.
+2.  Flagship adds the internship entry to the database.
 Use case ends.
```
Comment (jinyang628) [inline docs/DeveloperGuide.md:500]: Javin, should we use the `Flagship` markdown for all references to Flagship and the `user` mark down for all references to users. I'm not too sure myself

**Review by jinyang628 — DISMISSED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (javinchua) [reply docs/DeveloperGuide.md:500]: by markdown do u mean enclosing it in `user` , like u have done with ur comment? cause i referenced a few other UGs and they all seem to just put it as plain text. @jinyang628

Comment (jinyang628) [reply docs/DeveloperGuide.md:500]: Okay got it!

**Review by jinyang628 — COMMENTED** [review]
LGTM

**Review by s-kybound — APPROVED** [review]
Good. LGTM.

**Pull request merged**